eos_data_distribution/ndn/base.py

```diff
@@ -203,13 +203,13 @@ def _onData(self, interest, data):
         self._callbackCount += 1
         self.emit('data', interest, data)
 
-    def expressInterest(self, name=None, forever=False):
-        if name is None:
-            name = self.name
+    def expressInterest(self, interest=None, forever=False):
+        if interest is None:
+            interest = self.name
 
-        logger.debug("Express Interest name: %s", name)
-        onTimeout = partial(self.onTimeout, forever=forever, name=name)
-        self.pit[name] = self.face.expressInterest(name, self._onData, onTimeout)
+        logger.debug("Express Interest name: %s", interest)
+        onTimeout = partial(self.onTimeout, forever=forever, name=self.name)
```

Comment:
This doesn't make sense to me -- if `onTimeout` gets called, it will request another interest with the base name of the consumer, rather than the one that actually timed out. This seems wrong to me -- we should fix up `onTimeout`.